app/src/main/java/org/simple/clinic/newentry/PatientEntryLogic.kt

```diff
@@ -3,17 +3,24 @@ package org.simple.clinic.newentry
 import com.spotify.mobius.First
 import com.spotify.mobius.First.first
 import com.spotify.mobius.Next
+import com.spotify.mobius.Next.next
 import com.spotify.mobius.Next.noChange
 
 fun patientEntryInit(
     model: PatientEntryModel
-): First<PatientEntryModel, PatientEntryEffect> {
-  return first(model)
-}
+): First<PatientEntryModel, PatientEntryEffect> =
+    first(model, setOf(FetchPatientEntry))
```

Comment:
We won't need to emit the `FetchPatientEntry` on a view restoration event, right?

Reply:
We will at the moment. This is the same behavior we have in the code right now. That explains why we are using the Activity paused event to save the ongoing into the database and then we are re-querying it in the `ScreenCreated()` event.

Since we do have a model, we can persist it in the `Bundle` and restore it and avoid re-queries. I would still keep this behavior and change this after the migration by writing a test.

I have noted this and will do it once we delete the controller class.